beets/autotag/mb.py

```diff
@@ -416,18 +416,16 @@ def album_info(release):
         if reltype:
             info.albumtype = reltype.lower()
 
-    # Log the new-style "primary" and "secondary" release types.
-    # Eventually, we'd like to actually store this data, but we just log
-    # it for now to help understand the differences.
+    # Set the new-style "primary" and "secondary" release types.
     if 'primary-type' in release['release-group']:
         rel_primarytype = release['release-group']['primary-type']
         if rel_primarytype:
-            log.debug('primary MB release type: ' + rel_primarytype.lower())
+            info.albumtypes = rel_primarytype.lower()
     if 'secondary-type-list' in release['release-group']:
         if release['release-group']['secondary-type-list']:
-            log.debug('secondary MB release type(s): ' + ', '.join(
-                [secondarytype.lower() for secondarytype in
-                    release['release-group']['secondary-type-list']]))
+            info.albumtypes += ',' + ','.join(
```

Comment:
Using `+= ','` seems to run the risk of starting the string with a comma, if the album is missing a `primary-type`?

Maybe a simpler alternative would be to build up a list, `albumtypes`, that starts out empty and is then expanded under both `if` conditions, and then `join` it all together at the end.

Reply:
Fair point! I had thought of that, but forgot about it just as quickly.